config.json

```diff
@@ -0,0 +1,107 @@
+{
+  "verbose": true,
+  "debug": true,
+  "stats": true,
+  "exit-after": 0,
+  "split-output": false,
+  "recognize-tcp-sessions": false,
+  "http-pprof": "",
+  "input-dummy": null,
+  "OutputDummy": null,
+  "output-stdout": false,
+  "output-null": false,
+  "input-tcp": null,
+  "InputTCPConfig": {
+    "input-tcp-secure": false,
+    "input-tcp-certificate": "",
+    "input-tcp-certificate-key": ""
+  },
+  "output-tcp": null,
+  "OutputTCPConfig": {
+    "output-tcp-secure": false,
+    "output-tcp-sticky": false
+  },
+  "output-tcp-stats": false,
+  "input-file": null,
+  "input-file-loop": false,
+  "output-file": null,
+  "OutputFileConfig": {
+    "output-file-flush-interval": 1000000000,
+    "output-file-queue-limit": 256,
+    "output-file-append": false,
+    "output-file-buffer": "/tmp"
+  },
+  "input_raw": null,
+  "InputRAWConfig": {
+    "input-raw-engine": "libpcap",
+    "input-raw-track-response": false,
+    "input-raw-realip-header": "",
+    "input-raw-expire": 2000000000,
+    "input-raw-protocol": "http",
+    "input-raw-bpf-filter": "",
+    "input-raw-timestamp-type": "",
+    "input-raw-immediate-mode": false,
+    "BufferSize": 0,
+    "input-raw-override-snaplen": false,
+    "input-raw-buffer-size": "0"
+  },
+  "output-file-size-limit": "32mb",
+  "output-file-max-size-limit": "1TB",
+  "copy-buffer-size": "5mb",
+  "middleware": "",
+  "InputHTTP": null,
+  "output-http": null,
+  "prettify-http": false,
+  "OutputHTTPConfig": {
```

Comment:
I think we should use only 1 format for config variable. For example
1. Start from lower case
2. Separete with "-" or "_" symbols
3. Use only lower case ?